.github/actions/setup-nvidia/action.yml

```diff
@@ -260,3 +260,6 @@ runs:
           sudo nvidia-persistenced || true
           # This should show persistence mode ON
           nvidia-smi
+
+          # check if the container-toolkit is correctly installed and CUDA is available inside a container
+          docker run --rm -t --gpus=all python:3.13 nvidia-smi
```

Comment:
Given the volume of our CI I expect that this would run into rate limits sooner rather than later, whatever image we end up using here should either come from public ECR or from our own ECR

Reply:
```suggestion
          docker run --rm -t --gpus=all public.ecr.aws/docker/library/python:3.13 nvidia-smi
```